seq2science/rules/configuration_generic.smk

```diff
@@ -260,7 +260,13 @@ if "assembly" in samples:
     providers = PickleDict(providersfile)
     ftype = "annotation" if annotation_required else "genome"
     search_assemblies = [assembly for assembly in remote_assemblies if providers.get(assembly, {}).get(ftype) is None]
-
+    
+    # custom assemblies without provider for local/remote annotations
+    if "scrna_seq" in get_workflow() and 'kallistobus' in config['quantifier']:
+        if 'kite' in config['quantifier'].get('kallistobus').get('ref'):
```

Comment:
```
if "scrna_seq" in get_workflow():
    if 'kite' in config['quantifier'].get('kallistobus', {}).get('ref', ""):
```

`dict().get("key")` returns `None` if a default is unspecified. `.get()` and `in` wouldn't work :o